05_iigo/index.tex

```diff
@@ -467,16 +467,28 @@ \section{IIGO Session order}
 
 \section{Future Work}
 
-\begin{itemize}
-    \item \textbf{Diplomatic sanctions}: Although having the potential of being a good alternative for severer sanctions discussed in  Section~\ref{sec:sanctions}, diplomatic sanctions are \emph{not} implemented within the scope of the coursework. \\
+\textbf{Diplomatic sanctions.} Although having the potential of being a good alternative for severer sanctions discussed in  Section~\ref{sec:sanctions}, diplomatic sanctions are \emph{not} implemented within the scope of the coursework. \\
     Suggested diplomatic sanctions include:
         \begin{itemize}
             \item Revoking an island's eligibility to vote and to be elected for a position.
-            \item Revoking an island's eligibility to propose a rule/motion.
+            \item Revoking an island's eligibility to propose any rule or a specific rule.
         \end{itemize}
-    \item \textbf{Immutable rules}: Within the scope of the coursework, a subset of rules could have been categorised as immutable. This means that to change such immutable rules, the islands first need to vote to change their status to be \emph{mutable}, and consequently, hold another vote to change these mutable rules.
+\textbf{Immutable rules.} Within the scope of the coursework, a subset of rules could have been categorised as immutable. This means that to change such immutable rules, the islands first need to vote to change their status to be \emph{mutable}, and consequently, hold another vote to change these mutable rules. An example of such an immutable rule could have been "An island in the archipelago cannot hold multiple IIGO roles (i.e. the President, Speaker, and Judge) in the same turn if at least two other islands are still alive.". This immutable rule would allow islands to avoid the tyranny that might arise from the lack of separation of power in any turn.\\
     %\item \textbf{Adding rules to the proposal list: } 
-\end{itemize}
+\textbf{Scope of IIGO.} Within the scope of the coursework, additional rules could be defined to audit levels of interaction at the IITO, IIFO and foraging. \\
+    Suggested additional rules include:
+        \begin{itemize}
+            \item During an IITO session, each island is not \emph{permitted} to give gifts to a sanctioned island.
```

Comment:
```suggestion
            \item During an IITO session, each island is not \emph{permitted} to give gifts to a sanctioned island.
            \item Each island is \emph{obliged} to send the gift amount it agreed to send during the IIGO session. 
```

Reply:
no. IITO is a jungle. 
there are no 'obligations' in IITO

Reply:
there is not a _constitution_ in IITO which determnies the level of interactions there.

if you wanna use 'obligation' within IITO context, we should write it more clearly, not in just a **single** full sentence like this one

Reply:
I don't see why this could not be a rule agent can vote in and to as they please

Reply:
oh ok. you are right. :)
but than this should not be included here , but at a different section called
"possible additional rules" or smth.
wouldn't this be better?